### PR TITLE
Update to 0.75.1. Add README. Add blog post for 3.8.0

### PR DESCRIPTION
**Pull request by rnc**

### CODE REVIEW

**Comment by rnc** [comment]
@goldmann  Why is https://github.com/cekit/cekit.io/blob/master/config.toml#L38 marked as hidden please?

**Comment by goldmann** [comment]
@rnc We don't have a getting started page. It's probably a leftover from a template.

**Comment by rnc** [comment]
> @rnc We don't have a getting started page. It's probably a leftover from a template.

It _was_ (trying?!) to point to sections on e.g. https://docs.cekit.io/en/latest/getting-started/index.html In my opinion it should be enabled and fixed.

File: content/blog/2021/01/cekit-3.8.0-released.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: "CEKit 3.8.0 released"
+author: "Nick Cross"
+date: 2021-01-20
```
Comment (goldmann) [inline content/blog/2021/01/cekit-3.8.0-released.adoc]: Can you change the date and make it today?

Comment (rnc) [reply content/blog/2021/01/cekit-3.8.0-released.adoc]: Done

**Pull request merged**